sdk/storage/azure-storage-blob/azure/storage/blob/_shared/parser.py

```diff
@@ -18,3 +18,10 @@ def _str(value):
 
 def _to_utc_datetime(value):
     return value.strftime('%Y-%m-%dT%H:%M:%SZ')
+
+
+def get_empty_chunk(chunk_size):
```

Comment:
we can move the method out of _shared folder if you are fine with having separate download.py for file and blob

Reply:
You can create a  separate _download_helpers.py file just like we have one for uploads.

Reply:
using this chunk_data = b"\x00" * self.chunk_size instead in the next commit